core/src/i18n/pl.yaml

```diff
@@ -59,8 +56,6 @@ cron-jobs:
   title: Cron Jobs
 custom-resource-definitions:
   title: Definicje zasobów niestandardowych
-deployments:
-  title: Deploymenty
 discovery-and-network:
   title: Odkrywanie i Sieć
 functions:
```

Comment:
Re lines 74-75 (I can't add a comment there): 
you have two different translations for `completion`. I'd suggest you stick to one. 

Either
```
  completion-time: Czas ukończenia
  completions: Ukończenia
```
or
```
  completion-time: Czas realizacji
  completions: Realizacje
```